src/middlewared/middlewared/plugins/filesystem.py

```diff
@@ -138,3 +140,57 @@ def file_get_contents(self, path, options=None):
                 f.seek(options['offset'])
             data = binascii.b2a_base64(f.read(options.get('maxlen'))).decode().strip()
         return data
+
+
+class GetAndWatchEventSource(EventSource):
```

Comment:
I wonder if less generic and more clear name exists? Like `FollowTailEventSource` (like in `tail -f`)

Reply:
That indeed sounds better